texar_vocab.txt

```diff
@@ -0,0 +1,1000 @@
+no
```

Comment:
I think you don't have to commit this file.

Reply:
added the file to .gitignore